### ISSUE TITLE
Color Vision - Publish 1.2 with Sound Design and PhET-iO Upgrade

### ISSUE THREAD

**Issue by kathy-phet** [phetsims/color-vision#141]
Color Vision has been prioritized for upgrades with inclusive design team research. 

This issue will track PhET-iO design and development related work needed to be done in parallel with simulation upgrade and re-publication.

**Comment by jbphet** [comment]
For reference, the primary issue tracking the sound design implementation is #138.

**Comment by jbphet** [comment]
As of 8/30/2022, work on adding sound to this simulation is being paused.  I'm not sure if that means that we will continue with the phet-io work or not, but I thought it was important to note that there will not be much more work on the sound design for a while.

**Comment by jbphet** [comment]
I spoke with @kathy-phet about this, and she said that the phet-io version of this is also not a priority at this time, so I'm going to close this as "won't do" (or whatever the closest label is).
